Fixing the issue when build_grammar with BLOB files

Makopo opened the pull request:
I had no trouble when build locally. However, when I uploaded Academic.schema as-is and executed `kes.exe build_grammar`, the command failed with:

```
Input XML: https://xxxxxx.blob.core.windows.net/manual1/Academic.xml
Output Grammar: https://xxxxxx.blob.core.windows.net/manual1/Academic.grammar
ERROR: The grammar https://xxxxxx.blob.core.windows.net/manual1/Academic.xml imports the schema 'academic.schema', but it is not found at http://xxxxxx.blob.core.windows.net/manual1/academic.schema
```

This is the fix for the issue.

azurecla commented:
Hi __@yamatoya__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>You've already signed the contribution license agreement. Thanks!</span>
        <p>The agreement was validated by Microsoft and real humans are currently evaluating your PR.</p>

TTYL, AZPRBOT;

Makopo committed:
Fixing the issue when build_grammar with BLOB files

If you upload a file named 'Academic.schema' as a schema file on BLOB and run kas.exe, the command will fail with the following message:

ERROR: The grammar https://xxxxxx.blob.core.windows.net/manual1/Academic.xml imports the schema 'academic.schema', but it is not found at http://xxxxxx.blob.core.windows.net/manual1/academic.schema

This is the fix for the issue.
